src/components/hr/hr.component.tsx

```diff
@@ -19,6 +19,9 @@ export const Hr = ({
   const largeScreen = useIsAboveBreakpoint(adaptiveMxBreakpoint);
   let marginLeft = ml;
   let marginRight = mr;
+  // We have had to ignore the coverage for this if statement as it is not possible to test the adaptive behaviour in RTL.
+  // This is because RTL does not support the matchMedia API which is used to determine the screen size. Instead we are now testing this behaviour in Playwright.
+  /* istanbul ignore next */
```

Comment:
```suggestion
  /* istanbul ignore if */
```

I think it's worth us testing the else here in unit if possible and only ignore what we need to rather than it all